File: ch02/prime1.c
#include<stdio.h>

#define N 1000000

int main(void)
{
		int i, n;
		int cnt = 0;
		unsigned long counter = 0;
		for(n = 2; n <= N; n++){
				for(i = 2; i < n; i++){
						counter++;
						if (n % i == 0)
								break;
				}
				if (n == i){
						printf("%d ", n);
						cnt++;
				}
		}
		printf("나눗셈을 실행한 횟수 : %lu\n", counter);
		printf("%d까지의 소수의 갯수는 %d 입니다.\n", N, cnt);

		return 0;
}

File: ch01/python3/ch02/prime1.py
N = 100

if __name__ == '__main__':
    cnt = 0
    counter = 0
    for n in range(2, N+1):
        i = 0
        for i in range(2, n+1):
            counter += 1
            if n % i == 0:
                break
        if n == i:
            print("{} ".format(n), end='')
            cnt += 1
    print()
    print("나눗셈을 실행한 횟수 : {}".format(counter))
    print("{}까지의 소수의 갯수는 {} 입니다.".format(N, cnt))
